ansible/roles/daliuge_lofar/tasks/runtime.yml

```diff
@@ -0,0 +1,25 @@
+---
+# daliuge_lofar/tasks/runtime.yml
+#
+#
+- name: Install developments tools envrironment group
+  yum:
+    name: "@Development Tools"
+    state: installed
+
+- name: Ensure docker is installed for this use case
+  yum:
+    name: docker, python-pip, docker-compose, sysstat
+    state: installed
+
+- name: Enable and start docker
+  systemd:
+    name: docker
+    enabled: yes
+    state: started
+
+- name: Ensure selected packages are installed
+  yum:
+    name: "{{item}}"
```

Comment:
You can pass a list of packages in one move in this task, avoiding the use of with_items, which usually works out way quicker.

Reply:
Ansible will internally translate the with_items to a single comma-separated item to avoid this issue.